src/main/java/seedu/address/model/person/Name.java

```diff
@@ -34,6 +34,10 @@ public Name(String name) {
         fullName = name;
     }
 
+    public String getFullName() {
+        return fullName;
+    }
+
```

Comment:
i don't think this is necessary? since fullname is public already

Reply:
hmm but why is it public though

Reply:
from AB3 one thats why got some lines is like getName().fullname